src/auth/user.ts

```diff
@@ -36,6 +36,16 @@ export function useCurrentUser(name?: string) {
   return authUserMap.get(useFirebaseApp(name))!
 }
 
+/**
+ * Returns a boolean indicating if the current user has finished loading and is no longer undefined. This is useful for
+ * hiding navigation until the current user is loaded and we know if they are authenticated or not.
```

Comment:
Let's use this and rewrap around lines:


```suggestion
 * Helper that returns a computed boolean that becomes `true` as soon as the current user is no longer `undefined`. Note this doesn't ensure the user is logged in, only if the initial signing process has run.
```